src/components/formFields/input.tsx

```diff
@@ -67,9 +74,11 @@ export const InputDate: React.FC<IInputProps & IReactHookFormProps> = ({
   defaultValue,
   validation,
   register,
+  placeholder,
   errors,
 }) => (
   <TextField
+    placeholder={placeholder}
```

Comment:
This one doesn't work I think.

Note: I don't think we need a placeholder on a date input anyway, so just removing it should be OK.